.travis.yml

```diff
@@ -33,9 +33,7 @@ install:
   - export DATABASE_URL=postgres://localhost/dashboard
   - export DATABASE_POOL_SIZE=5
   - rustup default $(cat rust-toolchain)
-  - cargo install cargo-update || true
-  - cargo install-update-config diesel_cli --default-features false --feature postgres
-  - cargo install-update -i diesel_cli
+  - cargo install --no-default-features --features postgres --force diesel_cli
```

Comment:
Why was `cargo-install-update` removed?

Reply:
There is an error here: https://travis-ci.com/rust-lang/rfcbot-rs/builds/128457500#L371-L390
I tried cargo-install-update locally but the result are the same.

Reply:
Since we're using nightly anyway pass `-Zinstall-update` to `cargo install` then.